src/App.js

```diff
@@ -6,15 +6,17 @@ import ArticlesPage from "./Components/Articles/ArticlesPage";
 import HomePage from "./Components/HomePage";
 import LoginPage from "./Components/Login/LoginPage";
 import PrivateRoute from "./Components/PrivateRoute";
-import AddArticle from "./Components/Articles/AddArticle";
```

Comment:
It is better to use absolute imports instead of relative ones. check https://create-react-app.dev/docs/importing-a-component/#absolute-imports

Reply:
Done